taxi_service/settings.py

```diff
@@ -20,9 +20,8 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-8ovil3xu6=eaoqd#-#&ricv159p0pypoh5_lgm*)-dfcjqe=yc"
-)
+SECRET_KEY = ("django-insecure-8ovil3xu6=eaoqd#-"
+              "#&ricv159p0pypoh5_lgm*)-dfcjqe=yc")
 
 # SECURITY WARNING: don"t run with debug turned on in production!
 DEBUG = True
```

Comment:
The `DEBUG` setting is set to `True`, which is not suitable for production environments. Ensure that `DEBUG` is set to `False` in production to avoid exposing sensitive information.